lib/plugins/wallet/monerod/monerod.js

```diff
@@ -175,12 +156,20 @@ function newFunding (account, cryptoCode, settings, operatorId) {
         balanceRes,
         fetch('create_address', { account_index: 0 })
       ]))
-    .then(([balanceRes, addressRes]) => ({
-      fundingPendingBalance: BN(balanceRes.balance).minus(balanceRes.unlocked_balance),
-      fundingConfirmedBalance: BN(balanceRes.unlocked_balance),
-      fundingAddress: addressRes.address
-    }))
-    .catch(err => handleError(err))
+    .then(([balanceRes, addressRes]) => Promise.all([
+        balanceRes,
+        addressRes,
+        fetch('get_transfers', { pool: true, account_index: 0 })
+      ]))
+    .then(([balanceRes, addressRes, transferRes]) => {
+      const memPoolBalance = _.reduce((acc, value) => acc.plus(value.amount), BN(0), transferRes.pool)
+      return {
+        fundingPendingBalance: BN(balanceRes.balance).minus(balanceRes.unlocked_balance).plus(memPoolBalance),
```

Comment:
The docs aren't very clear about this, but doesn't `balance` include the amounts of the txs of the pool? I.e., doesn't this line account for them twice?

Description of the `balance` of the output of [`get_balance`](https://www.getmonero.org/resources/developer-guides/wallet-rpc.html#get_balance):

> The total balance of the current monero-wallet-rpc in session.

And description of `pool` of [`get_transfers`](https://www.getmonero.org/resources/developer-guides/wallet-rpc.html#get_transfers):

> Include transfers from the daemon's transaction pool.